add-on/src/popup/page-action/index.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+/* eslint-env browser, webextensions */
+
+const choo = require('choo')
+const pageActionPage = require('./page')
+const browserActionStore = require('../browser-action/store')
+
+const app = choo()
+
+// Reuse store to setup state defaults and event listeners for mutations
+app.use(browserActionStore)
+
+// Register our single route
+app.route('*', pageActionPage)
+
+// Start the application and render it to the given querySelector
+app.mount('#root')
```

Comment:
I know WebExtensions are not the easiest thing to test, even with manual human testing. I assume the interactions, including this `pageAction` addition, will be handled by the smoke tests in #276, IIUC?

Reply:
#276 is mostly for "checking current state of things" (Edge-related issue was spawned from it).  Proper one is #145.  But yes, for now we just need to eyeball things that can't be tested via mocha.

Some background:

In old SDK (`legacy-sdk` branch) we had very nice suite of tests that run in actual browser and utilized asserts against browser UI ([example](https://github.com/ipfs-shipyard/ipfs-companion/blob/legacy-sdk/test/test-gui.js))

There is no such test library for WebExtensions APIs, but I described the need in https://github.com/mozilla/web-ext/issues/5